pages.zh/linux/grub-install.md

```diff
@@ -0,0 +1,16 @@
+# grub-install
+
+> 安装 GRUB 到设备。
+> 更多信息：<https://www.gnu.org/software/grub/manual/grub/html_node/Installing-GRUB-using-grub_002dinstall.html>。
+
+- 安装 GRUB 到基于 BIOS 的系统：
+
+`grub-install --target={{i386-pc}} {{path/to/device}}`
```

Comment:
Please translate the arguments for get this easier for non-English speakers. Translate `path/to/device` and the other examples.

Reply:
Also maybe not device, use `block_device` since that what they are called.

Reply:
But that should be done in the original page, before we change any translations

Reply:
anyone using grub-install，must kown  what is `device`, we  assess that they know device like `/dev/sdx`,if they want kown detail info,they would refers to archlinux wiki .

Reply:
It would still be great if you could translate as many arguments as possible. Especially `path/to/device`.

Reply:
in legacy bios,the grub will be install  on disk between sector2-sector62,so the `device` must be sdX which means disk,not sdX1 which means partion,should i translate `path/to/device` into `/dev/sdX` directly? or just in chinese?